applications/launchpad/gui-react/src/components/Inputs/PasswordInput/index.tsx

```diff
@@ -21,11 +23,20 @@ import { PasswordInputProps } from './types'
 const PasswordInput = ({ ...props }: PasswordInputProps) => {
   const [showPassword, setShowPassword] = useState(false)
 
+  const { useReveal, useStrengthMeter } = props
+
+  const inputIcons = (
+    <InputIcons>
+      {useStrengthMeter ? <StrengthMeter password={props.value} /> : null}
+      {useReveal ? showPassword ? <Eye /> : <EyeSlash /> : null}
+    </InputIcons>
+  )
+
   return (
     <Input
       type={showPassword ? 'text' : 'password'}
       {...props}
-      inputIcon={showPassword ? <Eye /> : <EyeSlash />}
+      inputIcon={inputIcons}
```

Comment:
with this implementation, if only strength meter is used, you can still click it to show the password

is this behaviour on purpose? there is nothing in the issue that says it should work like that

Reply:
You are right! Fixed